core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -231,6 +252,10 @@ object StorageTool extends Logging {
       val metaPropertiesPath = Paths.get(directory, "meta.properties")
       val checkpoint = new BrokerMetadataCheckpoint(metaPropertiesPath.toFile)
       checkpoint.write(metaProperties.toProperties)
+
+      val bootstrapMetadata = BootstrapMetadata.create(metadataVersion)
```

Comment:
should we write this first, so that if it fails, we don't have a valid directory?

Reply:
The bootstrap file and meta.properties are in the same directory, so L254 should cover that

Reply:
ok